packages/ui/src/helpers/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { Logger } from 'aws-amplify';
+
+type LoggerCategory = 'Auth' | 'Geo' | 'Notifications' | 'Storage';
+
+export const getCategoryLogger = (category: LoggerCategory) =>
```

Comment:
super nit: `getCategoryLogger` -> `getLogger`

Reply:
Done in [b1796f0](https://github.com/aws-amplify/amplify-ui/pull/2885/commits/b1796f0bdddecbf9af3266324edbbfcf0c25fd3b)